save.go

```diff
@@ -118,7 +118,12 @@ func saveMain(_ *cobra.Command, args []string) error {
 func copySrc(src, dest string) error {
 	// Skip the .git directory for copying, if it exists, since we don't want to save the user's
 	// local Git config along with the source code.
-	opt := copy.Options{Skip: func(src string) (bool, error) { return strings.HasSuffix(src, ".git"), nil }}
+	opt := copy.Options{
+		Skip: func(src string) (bool, error) {
+			return strings.HasSuffix(src, ".git"), nil
+		},
+		AddPermission: 0700,
```

Comment:
I would imagine we don't need executable permissions on any of these files. Let's use 0600 instead?

Reply:
cool done